src/content/troubleshooting/_index.en.md

```diff
@@ -0,0 +1,135 @@
++++
+title = "Troubleshooting Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 5
+pre = "<b>1. </b>"
```

Comment:
This guide should not be number 1 (the first one) we show on the menu. I suggest to make it number 4. It also means that other pages would need to be updated so that we have the correct numbering:

1. Quickstart guide
2. Deployment
3. Architecture
4. Troubleshooting Guide
5. Releases
6. Online resources
7. Roadmap
8. Contributing